### ISSUE TITLE
Recommendations for training on a custom dataset

### ISSUE THREAD

**Issue by akshay-bapat-magna** [shanice-l/gdrnpp_bop2022#13]
Hello,
I am trying to train the algorithm on my custom dataset (50,000 images, all synthetically generated using PBR). I am currently stuck on training the YOLO-X model; training using the default settings resulted in a really terrible output model that cannot predict anything (see image). Do you have any suggestions for datasets or training parameters (e.g. number of images in the dataset, learning rate, LR scheduler type, etc.)?

In the image, all dark gray parts are to be detected and classified, the other TLESS parts are used as distractions and are supposed to be ignored.

Any kind of ideas/tips are welcome and appreciated.

![bboxes](https://user-images.githubusercontent.com/86367234/202527248-924541d5-946d-4f56-b930-503d834a6ffc.png)

**Comment by shanice-l** [comment]
You can visualize the ground-truth labels, and be careful about the bounding box format (XYXY or XYWH).

**Comment by shanice-l** [comment]
I think 50,000 images are enough. The problem lies somewhere else.

**Comment by monajalal** [comment]
@akshay-bapat-magna could you please share the script and tool that you used for creating custom data for GDRNpp? Thank you.
